data/json/items/comestibles/drink.json

```diff
@@ -1639,5 +1639,77 @@
     "phase": "liquid",
     "flags": [ "EATEN_COLD" ],
     "fun": 6
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "zerocola",
+    "looks_like": "cola",
+    "name": { "str_sp": "diet cola" },
+    "weight": "248 g",
+    "color": "brown",
+    "container": "can_drink",
+    "comestible_type": "DRINK",
+    "symbol": "~",
+    "quench": 24,
+    "healthy": -1,
+    "calories": 0,
+    "description": "Zero-calorie, caffeine-free cola with artificial sweeteners.  It has a weird aftertaste.  In times like these you would rather it had the precious calories.",
+    "price": 95,
+    "price_postapoc": 40,
+    "material": [ "junk" ],
+    "primary_material": "water",
+    "volume": "250 ml",
+    "charges": 1,
+    "phase": "liquid",
+    "flags": [ "EATEN_COLD" ],
+    "fun": 4
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "zeroorange",
+    "looks_like": "orangesoda",
+    "name": { "str_sp": "diet orange soda" },
+    "weight": "248 g",
+    "color": "yellow",
+    "container": "can_drink",
+    "comestible_type": "DRINK",
+    "symbol": "~",
+    "quench": 24,
+    "healthy": -1,
+    "calories": 0,
+    "description": "Zero-calorie, orange flavored diet soda.  Tastes even weirder because of the artificial sweeteners.  In times like these you would rather it had the precious calories.",
+    "price": 95,
+    "price_postapoc": 40,
+    "material": [ "junk" ],
+    "primary_material": "water",
+    "volume": "250 ml",
+    "charges": 1,
+    "phase": "liquid",
+    "flags": [ "EATEN_COLD" ],
+    "fun": 4
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "zerotropical",
+    "looks_like": "lemonlime",
+    "name": { "str_sp": "diet tropical soda" },
+    "weight": "248 g",
+    "color": "light_green",
+    "container": "can_drink",
+    "comestible_type": "DRINK",
+    "symbol": "~",
+    "quench": 24,
+    "healthy": -1,
+    "calories": 0,
+    "description": "Zero-calorie soda with pineapple, kiwi, and other tropical fruit flavorings.  Tastes even weirder because of the artificial sweeteners.  In times like these you would rather it had the precious calories.",
```

Comment:
I feel like we have an opportunity here to describe the artificial flavoring in a humorous way. Something like the fact that "it tastes just like pineapple or kiwi", to give an uneasy feeling that the flavoring is totally artifical and probably bad for you. I read a label on a real-world item once that said "contains nature-identical aromas!" (crudely translated to english from my language), just like they were advertising the fact that it was sort of based on nature, but really was not. How about we do the description here in the words of the manufacturer, instead of the thoughts of the player? Perhaps something like:
```
Zero calories! Enjoy the refreshing taste of the tropics without all the downsides of sugar! Zerotropical™ contains no additional additives or prohibited chemicals and its patented nature-identical aromas taste just like the real thing! Disclaimer: contains artificial sweeteners that may cause health problems.
```
Feel free to spin that to even more fun take on the artificial bit.

Reply:
Sucralose and aspartame, the two fake sugars most commonly used in the United States, are not harmful or unhealthy. These are popular misconceptions.

Reply:
Feels a bit long? I'll definitely think about something humorous though.

like maybe for `zerocola` - `Things go lighter with Cola Zero! No sugar, no caffeine, no worries!`

Reply:
I had a 'lil think, we could instead have a fourth soda variant, the Foodplace's Refreshing Drink(tm), with all the corpospeak satire.

`A curious entry to the diet soda market by Foodplace, the \"Refreshing Drink™\" tastes just like real human beverages and contains no additives proven to cause diabetes or heart diseases!`